pages/docs/manual/latest/installation.mdx

```diff
@@ -34,12 +34,62 @@ Alternatively, **to start a [ReasonReact](https://reasonml.github.io/reason-reac
 
 ## Integrate Into Existing JS Project
 
-You can install the toolchain locally to an existing JS project, through the familiar command:
+You can install ReScript locally into an existing JS project using the following steps.
```

Comment:
```suggestion
You can install ReScript locally into an existing JavaScript project using the following steps.
```

probably for this particular case, JavaScript is better.. but we use both versions